main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -210,8 +177,8 @@ object CompletionProvider {
       // through sortText
       //
       case _ => getExpCompletions() ++
-        getPredicateCompletions() ++
-        getTypeCompletions() ++
+        PredicateCompleter.getPredicateCompletions() ++
+        TypeCompleter.getTypeCompletions() ++
         getEffectCompletions()
```

Comment:
Instead of:

```
PredicateCompleter.getPredicateCompletions()
TypeCompleter.getTypeCompletions() 
```

maybe it would be more clean if the __methods__ had the same name:

```
PredicateCompleter.getCompletions(..)
TypeCompleter.getCompletions()
```

then we can introduce an interface for them. Maybe we should already introduce such an interface? Something like:

```
trait Completer {
  def getCompletions(...): ...
}
```

can you try that?